frontend/src/__tests__/actions/WindowActions.test.js

```diff
@@ -159,6 +160,8 @@ describe('WindowActions thunks', () => {
       });
     });
 
+    it.todo(`dispatches 'UPDATE_TABLE' action when tab is fetched`);
```

Comment:
Is this something that you want to fix later during the refactoring ?

Reply:
Yeah, as it's related but can wait.